### DISCUSSION TITLE
Sublime text auto-rename-tag

### DISCUSSION THREAD

**Discussion by yusuforzibekov** [community/community#71001] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

`Question`, `Recommendation`, `General`.

### Body

Hello! Does anyone know how to find an `auto-rename-tag` package for **Sublime Text?** As you may be aware, **Visual Studio Code** has this feature, but Sublime Text lacks `auto-rename-tag` functionality. Can you recommend another package with similar capabilities?
<br>
![image](https://res.cloudinary.com/practicaldev/image/fetch/s--iQPMJ7wW--/c_imagga_scale,f_auto,fl_progressive,h_900,q_66,w_1600/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/to2p7mgvqq39gvt0hkr9.gif)

**Comment by nurullon** [comment]
Hello!
Here's how to enable the auto-rename tag feature in Sublime Text:

Install Package Control: If you haven't already installed Package Control (a package manager for Sublime Text), you should install it. Follow the installation instructions here: https://packagecontrol.io/installation

Install AutoRenameTag Plugin:

Once you have Package Control installed, open Sublime Text.
Press Ctrl+Shift+P (Windows/Linux) or Cmd+Shift+P (macOS) to open the command palette.
Type "Package Control: Install Package" and press Enter.
In the input field, type "AutoRenameTag" and press Enter to install the package.
Usage:

Place your cursor inside an HTML or XML tag.
Press Alt+. (Windows/Linux) or Cmd+. (macOS) to automatically select the whole tag.
Start typing your new tag name, and it will replace the opening and closing tag names simultaneously.

**Reply by yusuforzibekov** [reply]
@nurullodev Hello but! there is no package named AutoRenameTag in sublime text 3. Thank you your reply.

**Comment by onlinebillexpert** [comment, marked as answer]
It seems like you're referring to a specific issue or request on GitHub related to the "Sublime Text auto-rename-tag" plugin, with the issue number #71001.

If you're looking for information or updates on this particular GitHub issue, you can follow these steps:

1. **Visit the GitHub Repository**: Go to the GitHub repository where the issue is posted. In this case, it's likely the repository for the "Sublime Text auto-rename-tag" plugin.

2. **Search for the Issue**: Once you're on the repository page, you can use the search feature to look for issue number #71001.

3. **Read the Issue**: Click on the issue with that number to read the details and discussions related to it. This will give you an understanding of what the issue is about and any updates or comments from the maintainers or contributors.

4. **Subscribe or Comment**: If you want to stay updated on the progress of this issue, you can subscribe to it by clicking the "Subscribe" button. You can also comment on the issue if you have additional information or if you want to express your interest in its resolution.

Remember that the progress and resolution of GitHub issues depend on the maintainers and contributors of the project, so it might take some time for updates to be made.
[Read](https://onlinebillexpert.com/)
